src/hb-ot-var-common.hh

```diff
@@ -249,6 +249,53 @@ struct TupleVariationHeader
   const TupleVariationHeader &get_next (unsigned axis_count) const
   { return StructAtOffset<TupleVariationHeader> (this, get_size (axis_count)); }
 
+  bool unpack_axis_tuples (unsigned axis_count,
```

Comment:
You can also return the hashmap directly. Seems like false is never returned.

Reply:
I'd still return a bool: there's a check on input shared_tuples and axis_count, if it's invalid, it'll return false